plugins/exec-cronjob.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: exec-cronjob
+spec:
+  version: v0.4.0
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/thecloudnatives/kubectl-plugins/archive/v0.4.0.zip
+    sha256: 688bceed5fdd54414fc70080dacb776025c9ab31fb7bda3fca166bba05976a38
+    files:
+    - from: kubectl-plugins-*/exec-cronjob/*
+      to: "."
+    bin: kubectl-exec_cronjob.sh
+  shortDescription: Execute a cron job
+  description: |
+    Execute a CronJob, by extracting the Job spec and apply it in the given
+    namespace
+  caveats: |
+    Usage:
+        kubectl exec-cronjob <name> [options]
+
+    Options:
+        -n, --namespace='': If present, the namespace scope for this CLI request
+        --dry-run: If true, only print the object that would be sent, without sending it.
+        -h, --help: Display this help
+
+    Read the documentation at:
+      https://github.com/thecloudnatives/kubectl-plugins
+  homepage: https://github.com/thecloudnatives/kubectl-plugins
```

Comment:
You are using one repo for multiple plugins, which is completely fine. But is there an anchor link to the docs for this particular plugin?